meetupapi/src/main/java/com/gdgistanbul/di/MeetupApiModule.kt

```diff
@@ -1,18 +1,32 @@
 package com.gdgistanbul.di
 
 import com.gdgistanbul.api.MeetupApi
+import com.gdgistanbul.api.MeetupSecureApi
 import org.koin.dsl.module
 import retrofit2.Retrofit
 import retrofit2.converter.moshi.MoshiConverterFactory
 
+const val baseUrlApi = "https://api.meetup.com/"
```

Comment:
I think, we should use capital letter when defining a const val like **BASE_URL_API**